src/main/kotlin/com/cjbooms/fabrikt/generators/GeneratorUtils.kt

```diff
@@ -74,7 +76,7 @@ object GeneratorUtils {
      * It resolves the schema for the given API operation. If multiple content medias are found, then it will
      * resolve to the schema reference of the first media type.
      */
-    fun RequestBody.toBodyRequestSchema(): List<Schema> =
+    private fun RequestBody.toBodyRequestSchema(): List<Schema> =
```

Comment:
**question (non-blocking):** I am not sure I completely understand to use case. 
In this MR, we only support inline-schema-generation if a `requestBody` is defined in the specs?

Reply:
Are you talking about the function as a whole or just my change?

Reply:
The entire MR actually. If i wanted to have this spec, it would not generate the enum. Is that correct? 
```
paths:
  /test:
    post:
      operationId: testPath
      parameters:
        - in: query
          name: testString
          required: true
          schema:
            schema:
              type: string
              enum:
                - A
                - B
```

Reply:
It would!

```kotlin
package playground.models

import com.fasterxml.jackson.annotation.JsonValue
import kotlin.String
import kotlin.collections.Map

enum class TestPostParametersTestString(
  @JsonValue
  val value: String
) {
  A("A"),

  B("B");

  companion object {
    private val mapping: Map<String, TestPostParametersTestString> =
        values().associateBy(TestPostParametersTestString::value)

    fun fromValue(value: String): TestPostParametersTestString? = mapping[value]
  }
}
```

```kotlin
package playground.controllers

import kotlin.Unit
import org.springframework.http.HttpStatus
import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Controller
import org.springframework.validation.annotation.Validated
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod
import org.springframework.web.bind.annotation.RequestParam
import playground.models.TestPostParametersTestString

@Controller
@Validated
@RequestMapping("")
interface TestController {
  /**
   *
   *
   * @param testString 
   */
  @RequestMapping(
    value = ["/test"],
    produces = [],
    method = [RequestMethod.POST]
  )
  fun testPath(@RequestParam(value = "testString", required = true)
      testString: TestPostParametersTestString): ResponseEntity<Unit>
}
```

Reply:
Do you have the branches merged somehow? If I am on `origin/issue-187` i don't get this code generated.

Reply:
No, just the branch as is.

What exactly are you doing? Did you create a new test directory and add that yaml spec of yours?

Reply:
Your schema file above is incorrect. Maybe errors are silently ignored and so nothing is generated?
Your operation is missing a response and you have doubled the `schema` tag.